src/Settings.php

```diff
@@ -227,14 +227,56 @@ public function __construct() {
 				'title'  => 'Cookies',
 				'fields' => [
 					[
-						'name'    => __( 'Cookies Text', 'planet4-master-theme-backend' ),
+						'name'    => __( 'Cookies Box General Text', 'planet4-master-theme-backend' ),
 						'id'      => 'cookies_field',
 						'type'    => 'wysiwyg',
 						'options' => [
 							'textarea_rows' => 5,
 							'media_buttons' => false,
 						],
 					],
+					[
+						'name' => __( 'Necessary Cookies label', 'planet4-master-theme-backend' ),
+						'id'   => 'necessary_cookies_label',
+						'type' => 'text',
+					],
+					[
+						'name'    => __( 'Necessary Cookies description', 'planet4-master-theme-backend' ),
+						'id'      => 'necessary_cookies_description',
+						'type'    => 'wysiwyg',
```

Comment:
Are we sure that the `wysiwyg` field always stores a string of text? Or might it also contain markup?

Reply:
I assume we don't want to allow custom formatting? There's some risk for bugs and security issues with outputting HTML tags from the database.

Reply:
For now I've only allowed italic and bold formatting, since these are allowed in the Cookies block too